<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1484</width>
    <height>1149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Korova</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0">
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1458</width>
         <height>527</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="scanTab">
       <attribute name="title">
        <string>Scan</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="5,5,1,50">
            <item row="0" column="0">
             <widget class="QRadioButton" name="previewRadioButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Preview (fast)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="gotoBeginningButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Beginning</string>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QSlider" name="bAdjustSlider">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QSlider" name="gAdjustSlider">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QComboBox" name="filmTypeComboBox">
              <item>
               <property name="text">
                <string>Auto (DXN)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Custom</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ilford FP4+</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ilford HP5+</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ilford Delta Pro 100</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ilford Delta Pro 400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ilford Pan F+</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kodak Portra 160</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kodak Portra 400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kodak Portra 800</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kodak Gold 200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kodak Ektar 100</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QSlider" name="rAdjustSlider">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="moveLeftButton">
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Film Type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="startStopPushButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Go to</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="filmColourComboBox">
              <item>
               <property name="text">
                <string>BW Negative</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Colour Negative</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BW Positive</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Colour Positive</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="gotoEndButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>End</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QSlider" name="wAdjustSlider">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Adjust</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="moveRightButton">
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="fullRadioButton">
              <property name="text">
               <string>Full</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>W</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>R</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>G</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>B</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Histogram</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QComboBox" name="histColourComboBox">
              <item>
               <property name="text">
                <string>All</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="histogramLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="exportTab">
       <attribute name="title">
        <string>Export</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="folderNameLineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Folder Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Include:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="fileResComboBox">
          <item>
           <property name="text">
            <string>4064DPI | 5760 x 3840 | 22MP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2032DPI | 2880 x 1920 | 5.5MP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1016DPI | 1440x960 | 1.3MP</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="fileFormatComboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>tiff</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>tiff</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bayer (raw)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>jpeg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>png</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="fileExportPushButton">
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="datetimeCheckBox">
            <property name="text">
             <string>Datetime</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="folderNameLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1484</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences_2"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionKorova"/>
    <addaction name="actionSoke_Eng"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSomething">
   <property name="text">
    <string>Something</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionPreferences_2">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionKorova">
   <property name="text">
    <string>Korova</string>
   </property>
  </action>
  <action name="actionSoke_Eng">
   <property name="text">
    <string>Soke Engineering</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
